src/codewhisperer/util/editorContext.ts

```diff
@@ -41,15 +45,12 @@ export function extractContextForCodeWhisperer(editor: vscode.TextEditor): codew
         )
     )
 
-    return {
-        filename: getFileNameForRequest(editor),
-        programmingLanguage: {
-            languageName:
-                runtimeLanguageContext.normalizeLanguage(editor.document.languageId) ?? editor.document.languageId,
-        },
-        leftFileContent: caretLeftFileContext,
-        rightFileContent: caretRightFileContext,
-    } as codewhispererClient.FileContext
+    return new CWFileContext(
+        getFileNameForRequest(editor),
+        runtimeLanguageContext.normalizeLanguage(editor.document.languageId) ?? 'plaintext',
```

Comment:
ideally, we shouldn't return early when `normalizeLanguage` return `undefined. 
will fix